src/components/AsideSection/AsideSection.test.tsx

```diff
@@ -45,13 +46,13 @@ describe('<AsideSection />', () => {
 
   describe('with customizer', () => {
     it('receives custom theme', () => {
-      const theme = { palette: {}, semanticColors: {} };
+      const theme = defaultTheme;
       const mountedComponent = mount(
         <Customizer settings={{ theme }}>
-          <AsideSection title="TITLE" />
+          <CustomizableAsideSection title="TITLE" />
         </Customizer>,
       );
```

Comment:
Would it be better to move these lines above to a `beforeEach` associated to the `with customizer` block?

Reply:
Yeah, that would be more consistent with other tests. Done.